### PR TITLE
checks and tests for CKV_AZURE_80, CKV_AZURE_81, CKV_AZURE_82, CKV_AZURE_83 

### PR DESCRIPTION
**Pull request by avivgu**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: checkov/terraform/checks/resource/azure/AppServiceDotnetFrameworkVersion.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from checkov.common.models.enums import CheckCategories
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+
+
+class AppServiceDotnetFrameworkVersion(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure that 'Net Framework' version is the latest, if used as a part of the web app (Manual)"
```
Comment (rotemavni) [inline checkov/terraform/checks/resource/azure/AppServiceDotnetFrameworkVersion.py:7]: @avivgu Can you please delete the `(Manual)`? it shouldn't have been in the spreasaheet

File: checkov/terraform/checks/resource/azure/AppServicePHPVersion.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class AppServicePHPVersion(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure that 'PHP version' is the latest, if used to run the web app (Manual)"
```
Comment (rotemavni) [inline checkov/terraform/checks/resource/azure/AppServicePHPVersion.py:7]: @avivgu same here

File: checkov/terraform/checks/resource/azure/AppServicePHPVersion.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        id = "CKV_AZURE_81"
+        supported_resources = ['azurerm_app_service']
+        categories = [CheckCategories.GENERAL_SECURITY]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf):
+        if 'site_config' in conf and 'php_version' in conf['site_config'][0]:
```
Comment (rotemavni) [inline checkov/terraform/checks/resource/azure/AppServicePHPVersion.py:14]: @avivgu This line can cause an exception if `'site_config' in conf` but `conf['site_config']` is not a list or it's length is 0. 
I would do
`if len(conf.get('site_config', [])) > 0 and 'php_version' in conf['site_config'][0]`

File: checkov/terraform/checks/resource/azure/AppServiceDotnetFrameworkVersion.py (hunk 1 above)
Comment (avivgu) [reply checkov/terraform/checks/resource/azure/AppServiceDotnetFrameworkVersion.py]: Done

File: checkov/terraform/checks/resource/azure/AppServiceJavaVersion.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class AppServiceJavaVersion(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure that 'Java version' is the latest, if used to run the web app (Manual)"
```
[the remark this answers is not in the document]
Comment (avivgu) [reply checkov/terraform/checks/resource/azure/AppServiceJavaVersion.py]: Done

File: checkov/terraform/checks/resource/azure/AppServicePHPVersion.py (hunk 2 above)
Comment (avivgu) [reply checkov/terraform/checks/resource/azure/AppServicePHPVersion.py]: Done

File: checkov/terraform/checks/resource/azure/AppServicePHPVersion.py (hunk 3 above)
Comment (avivgu) [reply checkov/terraform/checks/resource/azure/AppServicePHPVersion.py]: Done

**Pull request merged**